sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -22,7 +22,6 @@ init_config:
     #   - query: <QUERY>
     #     columns: <COLUMNS>
     #     tags: <TAGS>
-    #     collection_interval: <COLLECTION_INTERVAL>
```

Comment:
Why is this removed?

Reply:
One of the checks failed and asked me to run some script that aligns this file with config. We could also split in 2 PRs.